spec/classes/init_spec.rb

```diff
@@ -233,11 +232,11 @@
             end
 
             content = <<-END.gsub(%r{^\s+\|}, '')
```

Comment:
We can rely on heredoc now (since we dropped Ruby < 2.3)
```suggestion
            content = <<~END
```